r-package/grf/vignettes/llf.Rmd

```diff
@@ -0,0 +1,147 @@
+---
+title: "Introduction to local linear forests"
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{llf}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+```{r, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>"
+)
+set.seed(123)
+```
+
+```{r setup, warning = FALSE, message = FALSE}
+library(grf)
+library(glmnet)
+library(ggplot2)
+```
+
+This document aims to show how to implement local linear forests (LLF). We begin with the standard use case, walking through parameter choices and method details, and then discuss how to use local linear corrections with larger datasets. 
+
+```{r}
+n <- 600
+p <- 20
+sigma <- sqrt(20)
+
+mu <- function(x){
+  2 * x[1] + x[2] + x[3] * x[4]
+}
+
+X <- matrix(rnorm(n*p, 0, 1), nrow = n)
+Y <- apply(X, FUN = mu, MARGIN = 1) + sigma * rnorm(n)
+
+X.test <- matrix(rnorm(n*p, 0, 1), nrow = n)
+truth = apply(X.test, FUN = mu, MARGIN = 1)
```

Comment:
You have some`=` mixed in with `<-`.
Also, if you make section headings with `##` then a table of content will be displayed in the [documentation](https://grf-labs.github.io/grf/articles/diagnostics.html)

Reply:
Thanks for the suggestion! Let me think about that and try to put some nice visualizations in there. And I'll add section headers back in!